core/spec/models/spree/order_shipping_spec.rb

```diff
@@ -138,7 +138,7 @@ def emails
       before { Spree::OrderCancellations.new(order).short_ship([order.inventory_units.first]) }
 
       it "updates the order shipment state" do
-        expect { subject }.to change { order.reload.shipment_state }.from('ready').to('shipped')
+        expect { subject }.to change { order.reload.shipment_state }.from('pending').to('shipped')
```

Comment:
I think this might be an important behaviour change (and unfortunately this is the only spec that caught it).

A shipment which has some "canceled" inventory units should still be "ready" and shippable